### ISSUE TITLE
More keyboard shortcuts

### ISSUE THREAD

**Issue by deathau** [deathau/markdownload#59]
Or at least a keyboard shortcut for copy selection as markdown

**Comment by quintrino** [comment]
Is it possible to get a keyboard shortcut for the right click action "Download Tab as Markdown"?

**Comment by BrutalSimplicity** [comment]
Yes, a keyboard shortcut for copy markdown selection would be amazing. Are you open to consider PRs for this feature?

**Comment by deathau** [comment]
Another possibility is to ditch most of these commands and instead switch what the keyboard shortcut _does_ based on preferences (as suggested in #95)

**Comment by Flashwalker** [comment]
Looks like Chrome doesn't support custom shortcuts with **Shift**.
Just **Ctrl + Key** or **Alt + Key**. Shift is ignored.
